libs/design/src/scss/typography/sizes/_mixins.scss

```diff
@@ -1,66 +1,120 @@
-// Sets the font size to 52/64px
-@mixin heading-xl() {
+@mixin display-xl() {
 	@include embolden();
-	font-size: 3.25rem;
-	line-height: 1.1em;
-	letter-spacing: -1.5px;
+	font-size: 2.5rem;
+	line-height: 2.75rem;
 
 	@include breakpoint(tablet) {
-		font-size: 4rem;
+		font-size: 3.5rem;
+		line-height: 4rem;
 	}
 }
 
-// Sets the font size to 36/48px
-@mixin heading-lg() {
+.display-xl { /* stylelint-disable-line selector-class-pattern */
+	@include display-xl();
+}
+
+@mixin display-lg() {
 	@include embolden();
-	font-size: 2.25rem;
-	line-height: 1.1em;
-	letter-spacing: -0.5px;
+	font-size: 2rem;
+	line-height: 2.25rem;
 
 	@include breakpoint(tablet) {
 		font-size: 3rem;
+		line-height: 3.5rem;
 	}
 }
 
-// Sets the font size to 28/40px
-@mixin heading-md() {
-	@include embolden();
-	font-size: 1.75rem;
-	line-height: 1.1em;
-	letter-spacing: 0;
-
-	@include breakpoint(tablet) {
-		font-size: 2.5rem;
-	}
+.display-lg { /* stylelint-disable-line selector-class-pattern */
+	@include display-lg();
 }
 
-// Sets the font size to 24/32px
-@mixin heading-sm() {
+@mixin display-md() {
 	@include embolden();
 	font-size: 1.5rem;
-	line-height: 1.1em;
-	letter-spacing: 0;
+	line-height: 1.75rem;
 
 	@include breakpoint(tablet) {
 		font-size: 2rem;
+		line-height: 2.5rem;
 	}
 }
 
-// Sets the font size to 20/24px
-@mixin heading-xs() {
+.display-md { /* stylelint-disable-line selector-class-pattern */
+	@include display-md();
+}
+
+@mixin display-sm() {
 	@include embolden();
 	font-size: 1.25rem;
-	line-height: 1.1em;
-	letter-spacing: 0;
+	line-height: 1.5rem;
 
 	@include breakpoint(tablet) {
 		font-size: 1.5rem;
+		line-height: 2rem;
 	}
 }
 
+.display-sm { /* stylelint-disable-line selector-class-pattern */
```

Comment:
Don't expose these as global, it's against our api guidelines to expose global css classes.